pkg/domain/domain.go

```diff
@@ -2080,22 +2080,13 @@ func (do *Domain) PrivilegeHandle() *privileges.Handle {
 
 // BindHandle returns domain's bindHandle.
 func (do *Domain) BindHandle() bindinfo.GlobalBindingHandle {
-	v := do.bindHandle.Load()
-	if v == nil {
-		return nil
-	}
-	return v.(bindinfo.GlobalBindingHandle)
+	return do.bindHandle
 }
 
-// LoadBindInfoLoop create a goroutine loads BindInfo in a loop, it should
+// InitBindingHandle create a goroutine loads BindInfo in a loop, it should
 // be called only once in BootstrapSession.
-func (do *Domain) LoadBindInfoLoop(ctxForHandle sessionctx.Context, ctxForEvolve sessionctx.Context) error {
-	ctxForHandle.GetSessionVars().InRestrictedSQL = true
```

Comment:
just double check do we need to set `InRestrictedSQL`

Reply:
No need to set it here since now we're using an unified session pool, and all sctx from it are set with `InRestrictSQL = true`.